src/main/kotlin/no/nav/syfo/dialogmote/domain/DialogmotedeltakerArbeidsgiverVarsel.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.syfo.dialogmote.domain
+
+import no.nav.syfo.dialogmote.api.domain.DialogmotedeltakerArbeidsgiverVarselDTO
+import no.nav.syfo.varsel.MotedeltakerVarselType
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class DialogmotedeltakerArbeidsgiverVarsel(
```

Comment:
Hva er grunnen til at man ikke har felt `digitalt: Boolean` her slik som på DialogmotedeltakerArbeidstakerVarsel ?

Reply:
ArbeidsgiverVarsel vil være digitalt, både nå i starten og på sikt. For AG-varsel vil vi sende til nærmeste leder og/eller til Altinn basert på virksomhetsnummer.
Dette er i motsetning til AT-varsel hvor 1. versjon vil ha digitale varsler, men etterhvert må vi også sende brev(digitalt=false,)

Reply:
Skjønner 👍